Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.controller.ts

```diff
@@ -244,36 +233,16 @@
                 this.hasWriteAccess,
                 this.hasWriteAccess,
                 (query) => this.dataProcessingRegistrationService.getApplicableDataProcessors(this.dataProcessingRegistrationId, query, pageSize)
-                    .then(results => this.mapDataProcessingSearchResults(results)),
+                    .then(results => Helpers.Select2MappingHelper.mapDataProcessingSearchResults(results)),
                 null,
                 Helpers.Select2OptionsFormatHelper.formatOrganizationWithOptionalObjectContext
             );
         }
 
         private bindSubDataProcessors() {
-            const pageSize = 100;
-            this.bindingService.bindMultiSelectConfiguration<Models.DataProcessing.IDataProcessorDTO>(
-                config => this.subDataProcessors = config,
-                () => this.dataProcessingRegistration.subDataProcessors.sort((a, b) => a.name.localeCompare(b.name, Kitos.Shared.Localization.danishLocale)),
-                element => this.removeSubDataProcessor(element.id),
-                newElement => this.addSubDataProcessor(newElement),
-                this.hasWriteAccess,
-                this.hasWriteAccess,
-                (query) => this
-                    .dataProcessingRegistrationService
-                    .getApplicableSubDataProcessors(this.dataProcessingRegistrationId, query, pageSize)
-                    .then(results => this.mapDataProcessingSearchResults(results)),
-                null,
-                Helpers.Select2OptionsFormatHelper.formatOrganizationWithOptionalObjectContext
-            );
-        }
-
-        changeName(name) {
-            this.apiUseCaseFactory
-                .createUpdate("Navn", () => this.dataProcessingRegistrationService.rename(this.dataProcessingRegistrationId, name))
-                .executeAsync(nameChangeResponse => {
-                    this.headerName = name;
-                    return nameChangeResponse;
+            this.dataProcessingRegistrationService.get(this.dataProcessingRegistration.id)
```

Comment:
ok so you actually did some changes here... just call this one then in stead  of the loadState in the "closed" handler....